core/src/main/scala/kafka/server/ZkAdminManager.scala

```diff
@@ -93,7 +94,7 @@ class ZkAdminManager(val config: KafkaConfig,
     * Try to complete delayed topic operations with the request key
     */
   def tryCompleteDelayedTopicOperations(topic: String): Unit = {
-    val key = TopicKey(topic)
+    val key = new TopicKey(topic)
```

Comment:
No need for `new`. Ditto in a few other places below.

Reply:
Good point, I undid these changes